android/sentry.properties

```diff
@@ -1,4 +1,5 @@
 defaults.url=https://sentry.io/
 defaults.org=zooniverse-27
 defaults.project=mobile-app
-auth.token=906b01e443c44171a3fb2a9ef0bddb1d839fe8ce56634d04afe19ad1d5924258
+auth.token=See Instructions: https://github.com/zooniverse/mobile/wiki/Enabling-Sentry-for-your-Local-Builds
```

Comment:
Can we add this to the README? I think this is an important step when getting the app up and running with a new dev

Reply:
Totally! Lemme do that right now